pages/common/perl.md

```diff
@@ -21,3 +21,11 @@
 - Run a Perl script in debug mode, using `perldebug`:
 
 `perl -d {{script.pl}}`
+
+- Loo[p] over all lines of a file, editing them [i]n-place using a find/replace [e]xpression:
+
+`perl -p -i -e 's/{{find}}/{{replace}}/g' {{filename}}`
+
+- Replace all occurrences of a string in a file with another string, saving the original file with a given extension:
+
+`perl -p -i'.old' -e 's/{{find}}/{{replace}}/g' {{filename}}`
```

Comment:
@agnivade just to be sure, are you ok with this example's description not following the same structure as the one before it? I think extending the previous one would make it excessively long (in fact, I would even propose swapping
`Replace all occurrences of a string in a file with another string`
with
`Edit a file in-place using a find/replace expression`
or the even shorter
`Run a find/replace expression on a file`
which, when joined with the rest of the sentence, wouldn't make the description more reasonably sized :)